packages/wrangler/src/deployment-bundle/worker.ts

```diff
@@ -285,3 +285,50 @@ export function identifyD1BindingsAsBeta(
 export function removeD1BetaPrefix(binding: D1PrefixedBinding): string {
 	return binding.slice(D1_BETA_PREFIX.length);
 }
+
+// Types returned by the /script/{name}/references API
```

Comment:
Given that they are only used in the `delete.ts` file, there is no need to have these types here.
In fact, going forward, the aim is that this file describes what is uploaded as part of a deployment, rather than responses from API requests.
Can we move them into `delete.ts`, please?